x-pack/plugins/maps/public/lens/choropleth_chart/expression_renderer.tsx

```diff
@@ -48,6 +63,18 @@ export function getExpressionRenderer(coreSetup: CoreSetup<MapsPluginStartDepend
         // ignore error, lack of EMS file layers will be surfaced in dimension editor
       }
 
+      const renderComplete = () => {
+        const originatingApp = extractOriginatingApp(handlers.getExecutionContext());
+
+        if (originatingApp) {
+          plugins.usageCollection?.reportUiCounter(originatingApp, METRIC_TYPE.COUNT, [
+            `render_${originatingApp}_${'map'}`,
```

Comment:
how about just `render_${originatingApp}_map`. There is no need to wrap 'map' in an expression since its just a string

Reply:
thank you =)